### DISCUSSION TITLE
Capture Email and send to MAILCHIMP list when user registers for Free Trial

### DISCUSSION THREAD

**Discussion by FrazerHainsworth** [Glench/ExtPay#98] [Q&A]
Hi,

I am not a Developer but I am interested in using this for my extension and recommending that my developer uses it. I wanted to see if these things are possible.

I want users to have a 30 Day free trial

When they register their email with extentionpay I also want that email added to my MAILCHIMP email list via API as I don't want the user to need to enter their email 2 times.

I want to be able to manually upgrade some users accounts to PAID as I want to offer the extension as a package with another digital product so I will need to be able to give them a FREE account. I will also need to be able to give Free accounts to affiliates.

Also a new feature I would like to see is the option for the user to choose a monthly subscription or a annual subscription.

Thanks

Frazer Hainsworth
(www.TeachersXchange.com , www.ESLcourseware.com)

**Comment by BowTiedSwan** [comment]
You should be able to do that in two ways:

1. Go to the Developers tab inside your Stripe account. You can use the webhooks or the REST Api to create an integration.
2. EASIEST: Go to Zapier.com, create a hook when on new Stripe subscription -> creates new member in your email list:
![image](https://user-images.githubusercontent.com/86532747/228539925-51dfd80c-3332-4316-aecd-1899ba6a7fef.png)

**Reply by burkybang** [reply]
Have you actually tried this? I've had issues setting up Stripe integrations since my Stripe account is managed by ExtensionPay. Whenever I authenticate to Stripe from another platform, it automatically creates a new company in Stripe.

**Reply by BowTiedSwan** [reply]
Yes, just did now. Created a zap so whenever I get a new stripe user it subscribes it to a Convertkit form segment.

 Inside ExtPay go to "Free trial user dashboard" inside one of your plans:
![image](https://user-images.githubusercontent.com/86532747/228657838-9f65edc0-b32c-4232-9a6d-e6073c70ceda.png)

That will open your stripe's account specific store. When you are creating the Zap you have to select that store once you log into your Stripe account.
